include/arkode/arkode_lsrkstep.h

```diff
@@ -0,0 +1,75 @@
+/* -----------------------------------------------------------------
+ * Programmer(s): Daniel R. Reynolds @ SMU
+ * -----------------------------------------------------------------
+ * SUNDIALS Copyright Start
+ * Copyright (c) 2002-2024, Lawrence Livermore National Security
+ * and Southern Methodist University.
+ * All rights reserved.
+ *
+ * See the top-level LICENSE and NOTICE files for details.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SUNDIALS Copyright End
+ * -----------------------------------------------------------------
+ * This is the header file for the ARKODE LSRKStep module.
+ * -----------------------------------------------------------------*/
+
+#ifndef _LSRKSTEP_H
+#define _LSRKSTEP_H
+
+#include <arkode/arkode.h>
+
+#ifdef __cplusplus /* wrapper to enable C++ usage */
+extern "C" {
+#endif
+
+typedef int (*ARKSprFn)(sunrealtype t, sunrealtype* extsprad, void* user_data);
+
+/* ------------------
+ * LSRKStep Constants
+ * ------------------ */
+
+/* -------------------
+ * Exported Functions
+ * ------------------- */
+
+/* Creation and Reinitialization functions */
+
+SUNDIALS_EXPORT void* LSRKStepCreate(ARKRhsFn fe, ARKRhsFn fi, sunrealtype t0,
+                                     N_Vector y0, SUNContext sunctx);
+
+SUNDIALS_EXPORT int LSRKodeSetMethod(void* arkode_mem, int method);
```

Comment:
The valid integer values for the `method` input to this function should be specified by "enum" constants in this header file.  This helps ensure that users understand the valid inputs, and also to ensure that they do not specify an illegal value.